src/autotype/mac/AutoTypeMac.cpp

```diff
@@ -488,3 +493,17 @@ void AutoTypeExecutorMac::execKey(AutoTypeKey* action)
     m_platform->sendKey(action->key, false);
     usleep(25 * 1000);
 }
+
+void AutoTypeExecutorMac::execClearField(AutoTypeClearField* action = nullptr)
+{
+    Q_UNUSED(action);
+
+    m_platform->sendKey(Qt::Key_Control, true, true);
+    m_platform->sendKey(Qt::Key_A, true, true);
+    m_platform->sendKey(Qt::Key_Control, false, true);
```

Comment:
Look at the example here: https://developer.apple.com/reference/coregraphics/1456564-cgeventcreatekeyboardevent